pkg/common/helpers.go

```diff
@@ -49,3 +64,67 @@ func IsExpired(startTime int64, ttlTime int64) bool {
 	}
 	return false
 }
+
+// NewWFClient -- declare new workflow client
+func NewWFClient(cfg *rest.Config) wfv1versioned.Interface {
+	cli, err := wfv1versioned.NewForConfig(cfg)
+	if err != nil {
+		panic(err)
+	}
+	return cli
+}
+
+// NewAddonClient - declare new addon client
+func NewAddonClient(cfg *rest.Config) addonv1versioned.Interface {
+	cli, err := addonv1versioned.NewForConfig(cfg)
+	if err != nil {
+		panic(err)
```

Comment:
We do not want `panic()` in this code base, we also don't need these helpers since you can have dependency injection instead one liner, there's no re-use of the methods. Let's remove these.

Reply:
Do you mind share why you do no like panic()  ? panic, exit() generally indicates component does not work w/o it. Which exactly the case ?
Also, a basic function encapsulation makes developer easy to understand, which is a balance.

Reply:
f91a53ffc8d59d0bc3891a8d494fa2ed5eafb406